rental.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+require './book'
+
+class Rental
+  attr_reader :book, :person
```

Comment:
- [ ] Please add Getters and Setters for `@book` and `@person` like you did with `@date` because all of them are `instance variables`.